bin/git-psykorebase

```diff
@@ -19,7 +19,7 @@ function usage()
     echo "  -c|--continue:		Continue after the user updates conflicts."
 }
 
-while [[ $# > 0 ]]
+while [[ $# -gt 0 ]]
```

Comment:
Why was this changed? `>` is be perfectly and expressive find within `[[ ... ]]`, isn't it?

Reply:
`>` and `<` perform [lexicographical comparison](https://www.shellcheck.net/wiki/SC2071) rather than a numerical comparison in this context, so it was changed to `-gt`

Reply:
I see. Thx for the explanation. With this new knowledge I now have to have a look at some of my own scripts as well :)